node.go

```diff
@@ -78,8 +80,24 @@ type ClusterInfo struct {
 	Size int
 }
 
+// LogPositionHandler is used to interrogate the state of the log.
+type LogPositionHandler interface {
```

Comment:
Not sold on the name. Again, Log may not be the best choice here, but not sure what to call it yet.